File: cs50x/pset6/sentimental-mario-less/mario.py
from cs50 import get_int

# While loop to prompt user for integer input that keeps running until integer 1-8 is provided
while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

# For loop printing n rows of pyramid
for i in range(n):
    print(" " * (n - i - 1) + "#" * (i + 1))

File: cs50x/pset1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Name variable
    int n;
    do
    {
        //Ask for height input
        n = get_int("Height: ");
    }
    //Keep running height loop for integer input less than 1 or greater 8
    while (n < 1 || n > 8);
    //Add "i" rows for "i" less than "n"
    for (int i = 0; i < n; i++)
    {
        for (int k = n - i - 1; k > 0; k--)
            //Add n-k spaces for each row where k is greater than 0
        {
            printf(" ");
        }
        //Add j hashtags for each row where j is less than or equal to n
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        //Add line break
        printf("\n");
    }
}